PackagingProjects/CSharpPluginPayload/CSharpPluginPayload.csproj

```diff
@@ -53,6 +53,7 @@
     <FilesToCopy Include="$(SourcesRoot)\SonarQube.TeamBuild.Integration\bin\$(Configuration)\SonarQube.TeamBuild.Integration.dll" />
     <FilesToCopy Include="$(SourcesRoot)\SonarQube.MSBuild.Tasks\bin\$(Configuration)\SonarQube.MSBuild.Tasks.dll" />
     <TargetsToCopy Include="$(SourcesRoot)\SonarQube.MSBuild.Tasks\bin\$(Configuration)\Targets\SonarQube.Integration.targets" />
+    <TargetsToCopy Include="$(SourcesRoot)\SonarQube.MSBuild.Tasks\bin\$(Configuration)\Targets\SonarQube.Integration.ImportBefore.targets" />
```

Comment:
Note that the targets are placed in a "Targets" directory which is at the same level as the preprocessor (as opposed to the status quo where they would sit at the same level as the bootstrapper).